docs/authentication/3id-did/provider.md

```diff
@@ -0,0 +1,105 @@
+# 3ID DID Provider
```

Comment:
I don't feel well equipped to review this page. I'll defer to @oed here

Reply:
@oed have you had a chance to look at the 3id-did-provider page?